src/lwpMediaDevices.js

```diff
@@ -1267,7 +1270,7 @@ export default class extends lwpRenderer {
       name: this._getDeviceName(device),
       trackKind: this._deviceKindtoTrackKind(device.kind),
       connected: true,
-      groupId: device.groupId
+      groupId: device.groupId,
```

Comment:
Can you remove this comma at the end? haha

Reply:
Done :shipit: